entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/query/OCondition.kt

```diff
@@ -117,11 +117,21 @@ class OEdgeExistsCondition(
     }
 }
 
+class OSingleVertexCondition(
+    val rid:String
```

Comment:
Formatting. Please, apply for all the files.
```suggestion
    val rid: String
```